package.json

```diff
@@ -66,6 +67,7 @@
     "babel-preset-react-app": "^7.0.2",
     "bull": "^3.12.1",
     "bullmq": "^1.6.6",
+    "cpx": "^1.5.0",
```

Comment:
Couldn't find any documentation that this indeed supports windows, have you tried it yourself?

Also the docs state

> Copy file globs, watching for changes.
> 
> This module provides a CLI tool like cp, but with watching.

We're not really interested in watching for changes - could this have unforeseen side-effects? Would perhaps be better with a more plain file-copy lib.

Reply:
Actually i did not test it on Windows. I will test it next time 👍